test/controllers/income/estimatedPay/update/IncomeUpdateCalculatorControllerSpec.scala

```diff
@@ -938,10 +1221,14 @@ class IncomeUpdateCalculatorControllerSpec
 
     "redirect to sameEstimatedPay page" when {
       "the pay is the same" in {
-        val currentCache = Map(UpdateIncome_ConfirmedNewAmountKey -> "100")
-        val testController = createTestController(currentCache)
 
-        val result = testController.estimatedPayPage()(RequestBuilder.buildFakeRequestWithAuth("GET"))
+        val harness = UnifiedHarness.setupEstimatedPayPage()
+
+        when(journeyCacheService.currentCache(any()))
+          .thenReturn(Future.successful(Map(UpdateIncome_ConfirmedNewAmountKey -> "100")))
+
+        val result = harness.estimatedPayPage(RequestBuilder.buildFakeRequestWithAuth("GET"))
```

Comment:
If you add the suggested `postRequest` method to the request builder you could also add a `getRequest` method as well